lib/ui/surveydetail/survey_detail_screen.dart

```diff
@@ -126,16 +150,116 @@ class SurveyDetailScreenState extends ConsumerState<SurveyDetailScreen> {
         : const SizedBox.shrink();
   }
 
+  Widget _buildActionButton(BuildContext context, int index) {
+    if (index == 0) {
+      return _buildStartSurveyButton(context);
+    } else {
+      final isLastPage = index == pages.length;
+      if (isLastPage) {
+        return _buildSubmitButton();
+      } else {
+        return _buildNextButton();
+      }
+    }
+  }
+
+  Widget _buildStartSurveyButton(BuildContext context) {
+    return TextButton(
+      key: SurveyDetailScreenKey.btStart,
+      style: ButtonStyle(
```

Comment:
Let's use `TextButton.styleFrom(`

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/58/commits/c3e28b47d8e45884f706677f35777c7eb264e6f0